### PR TITLE
feat: add helm chart for docker registry secrets

### PR DESCRIPTION
**Pull request by smerle33**

### CODE REVIEW

File: charts/docker-registry-secrets/templates/registry-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{{- range .Values.imageCredentials.namespaces -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dockerhub-credential
+  namespace: {{ . }}
+type: kubernetes.io/dockerconfigjson
+data:
+  .dockerconfigjson: {{ template "imagePullSecret" $ }}
+---
```
[the remark this answers is not in the document]
Comment (smerle33) [reply charts/docker-registry-secrets/templates/registry-secret.yaml:12]: sounds better to me down than up if it's optional for you I'd prefer leave it that way

[the remark this answers is not in the document]
Comment (smerle33) [reply charts/docker-registry-secrets/templates/registry-secret.yaml:12]: sounds better to me bottom than up if it's optional for you I'd prefer leave it that way

[the remark this answers is not in the document]
Comment (dduportal) [reply charts/docker-registry-secrets/templates/registry-secret.yaml:12]: Given the official [YAML 1.2 specification](https://yaml.org/spec/1.2.2/), the `---` means a "separation between directives and document content", e.g. "document header" (as we do not have any YAML directives).

So my vote is in favor of the YAML syntax: `---` moved to the beginning.

[the remark this answers is not in the document]
Comment (lemeurherve) [reply charts/docker-registry-secrets/templates/registry-secret.yaml:12]: Also, there will be a yamllint error if it's at the bottom afaik.

**Comment by lemeurherve** [comment]
For ref: https://github.com/codecentric/helm-charts/pull/514

**Review by dduportal — APPROVED** [review]
Approving and merging: it will deploy a first version of this helm chart (`0.1.0`) as it looks good.

If we see that it does not work has expected or if we need more rework, we'll open new PR and deploy new versions ("release fast and often" principle).

Rationale is that we do not have "integration tests" for individual helm charts for now.

**Pull request merged**